src/services/postService.js

```diff
@@ -18,6 +18,21 @@ const {
 const VIDEO_URLS   = require('../utils/videoUrls');
 
 module.exports = {
+  broadcastAndStorePost: function({ authenticatedUserInstance, permLink, title, body, tags }) {
+    return authenticatedUserInstance.broadcastPost({ permLink, title, body, tags })
+      .then(broadcastSuccess => {
```

Comment:
This could be
```
.then(broadcastSuccess => broadcastSuccess ? authenticatedUserInstance.userInOurDb : null)
```

Reply:
i think we decided to change this to an if stantement in a previous PR